index.bs

```diff
@@ -522,27 +522,27 @@ This Section defines a number of new set of <a>Constrainable Properties</a> for
 
 <pre class="idl">
   partial dictionary MediaTrackConstraintSet {
-    ConstrainDOMString whiteBalanceMode;
-    ConstrainDOMString exposureMode;
-    ConstrainDOMString focusMode;
-    ConstrainPoint2D   pointsOfInterest;
-
-    ConstrainDouble    exposureCompensation;
-    ConstrainDouble    exposureTime;
-    ConstrainDouble    colorTemperature;
-    ConstrainDouble    iso;
-
-    ConstrainDouble    brightness;
-    ConstrainDouble    contrast;
-    ConstrainDouble    saturation;
-    ConstrainDouble    sharpness;
-
-    ConstrainDouble    focusDistance;
-    ConstrainDouble    pan;
-    ConstrainDouble    tilt;
-    ConstrainDouble    zoom;
-
-    ConstrainBoolean   torch;
+    ConstrainDOMString           whiteBalanceMode;
+    ConstrainDOMString           exposureMode;
+    ConstrainDOMString           focusMode;
+    ConstrainPoint2D             pointsOfInterest;
+
+    ConstrainDouble              exposureCompensation;
+    ConstrainDouble              exposureTime;
+    ConstrainDouble              colorTemperature;
+    ConstrainDouble              iso;
+
+    ConstrainDouble              brightness;
+    ConstrainDouble              contrast;
+    ConstrainDouble              saturation;
+    ConstrainDouble              sharpness;
+
+    ConstrainDouble              focusDistance;
+    (boolean or ConstrainDouble) pan;
```

Comment:
Can you walk through the rationale of this `boolean or ConstrainDouble`?
Why introducing this pattern now?

Reply:
My rationale is to keep similarity with audio/video. So while it is possible to say `navigator.mediaDevices.getUserMedia({video: {}})`, it is maybe clearer to say `navigator.mediaDevices.getUserMedia({video: true})`. Similarly while should be possible to say `navigator.mediaDevices.getUserMedia({video: {pan: {}, zoom: {}}})`, it might be clearer to say `navigator.mediaDevices.getUserMedia({video: {pan: true, zoom: true}})`.

Reply:
I agree with you about the simplicity. It looks much better.
I'm wondering why this pattern wasn't used before. Maybe there's something we're missing.

Reply:
@yellowdoge maybe would know

Reply:
@beaufortfrancois no particular reason, IIRC this spec predates the major clean up and homogeneisation of the constraints about 3-4y ago...? So perhaps it was just due an overhaul. IMHO this looks better & we should create implementation bugs.

Reply:
Thank you!